include/timer.h

```diff
@@ -1,5 +1,23 @@
 #ifndef _TIMER_H
 #define _TIMER_H
+#include "common.h"
 
+#define MAX_TIMER   128
+
+#define UNUSE   0
+#define USED    1
+
+typedef struct TIMER {
+    struct TIMER *next;
```

Comment:
Do we really need the next pointer? As I understand we have a pool of those rather than a linked list.

Reply:
Sorry, I seems like can't recall what it does mean, :-) the code is long far away ago. The next pointer is not for a link list. might be mean where is the next fired.